### DISCUSSION TITLE
【求助】2手14pro被骗子锁住有绳匠帮忙吗

### DISCUSSION THREAD

**Discussion by daofeng9612** [share121/inter-knot#2331] [其他]
骗子要求支付2000丁尼才给我开锁
![idlefish-msg-1723705769406 png](https://github.com/user-attachments/assets/08f1846a-c840-4e09-89ec-810ee6aaf4b2)

**Comment by NekoWise** [comment]


**Reply by daofeng9612** [reply]
则么操作

**Reply by daofeng9612** [reply]
可以的话加下联系方式微信号:shaoen09331谢谢绳匠
